src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -14,6 +16,24 @@
  */
 class Payments_Tab extends tad_DI52_ServiceProvider {
 
+	/**
+	 * Slug for the tab.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $slug = 'payments';
+
+	/**
+	 * Option key prefix for page creation flag.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_page_created = 'tec_tc_payments_page_created_';
```

Comment:
No need to make into a prefix, if the meta exists we consider that it was a automatically created page.

```suggestion
	public static $option_page_created_meta_key = 'tec_tc_payments_page_created';
```